Assets/Scripts/CandyAnimationsController.cs

```diff
@@ -62,8 +69,10 @@ public IEnumerator RotateMatchingCandies(GameObject candy, float duration, int r
     }
     public List<GameObject> CreateRotationList( List<GameObject> matchesHor, List<GameObject> matchesVer, GameSettings gameSettings, CandyPool candyPool)
     {
-        List<GameObject> combinedLists = new List<GameObject>();
-        List<GameObject> clonedLists = new List<GameObject>();
+        //List<GameObject> combinedLists = new List<GameObject>();
```

Comment:
It's a good habit to always remove code instead of commenting. they will add up and occupy your classes and make it harder to read and think about the implementation

Reply:
You're right, I often leave them because, if a new refactor doesn't work, I can easily go back and restart from scratch.

Reply:
That makes sense I also do that. But once you have a working solution and you want to push it to the repo it doesn't make sense to keep the comments 😅